### PR TITLE
Add studio backup section

### PR DESCRIPTION
**Pull request by Eggflaw**
Related to issue [#406](https://github.com/vinegarhq/vinegar/issues/406).

### CODE REVIEW

File: src/Configuration/tips-and-tricks/index.md [hunk 1]
```diff
@@ -32,3 +32,18 @@ See [Bloxstrap's guide to FastFlags](https://github.com/pizzaboxer/bloxstrap/wik
 - `WINEESYNC`: allows Wine Staging to use Esync, please see [HowToEsync](https://github.com/lutris/docs/blob/master/HowToEsync.md) for more information.
 - `WINEDEBUG`: for performance reasons, this has been set to `-all`, which disables most of the logging. In order to debug crashes of Wine, it is recommended to set this to an empty string (`""`).
 - `DXVK_HUD`: is a variable used by DXVK for a hud, for more information about it you can see the [DXVK README](https://github.com/doitsujin/dxvk#hud), which includes various other variables that can be set. This can be set, for example, to `fps` to display an fps counter.
+
+## Studio backup
+
+Before you delete your wineprefixes via `vinegar delete` or `flatpak run org.vinegarhq.Vinegar delete`, you should backup data you have on your Studio prefix. This includes your settings, files you have stored etc.
```
Comment (jrelvas-ipc) [inline src/Configuration/tips-and-tricks/index.md:38]: @apprehensions Is Studio saving local files on the prefix? We might want to change that.

--

In this case, I don't think the `etc.` is needed as there's actually no other kinds of data, so I'd word it more like:
```suggestion
Before you delete your wineprefixes via `vinegar delete` or `flatpak run org.vinegarhq.Vinegar delete`, you should backup data you have on your Studio prefix. This includes your settings and files you've stored.
```

File: src/Configuration/tips-and-tricks/index.md [hunk 2]
```diff
@@ -32,3 +32,18 @@ See [Bloxstrap's guide to FastFlags](https://github.com/pizzaboxer/bloxstrap/wik
...
+The Studio prefix location depends on how you installed Vinegar:
+
+- On Flatpak, it's located in `~/.var/app/org.vinegarhq.Vinegar/data/vinegar/prefixes/studio`
+- On other methods, it's located in `~/.local/share/vinegar/prefixes/studio`
+
+Where Roblox Studio store their settings and what they store:
+
+- `(studio prefix)/user.reg` : Studio theme, Docking layout
+- `(studio prefix)/drive_c/users/(username)/AppData/Local/Roblox/GlobalSettings_13.xml`: Display language, Fonts etc 
+- `(studio prefix)/drive_c/users/(username)/AppData/Local/Roblox/GlobalBasicSettings_13_Studio.xml`: MicroProfiler, Fullscreen etc
```
Comment (jrelvas-ipc) [inline src/Configuration/tips-and-tricks/index.md:49]: Some fixes to text consistency needed:
```suggestion
- `(studio prefix)/user.reg`: Studio theme, Docking layout.
- `(studio prefix)/drive_c/users/(username)/AppData/Local/Roblox/GlobalSettings_13.xml`: Display language, Fonts, etc. 
- `(studio prefix)/drive_c/users/(username)/AppData/Local/Roblox/GlobalBasicSettings_13_Studio.xml`: MicroProfiler, Fullscreen, etc.
```

**Review by jrelvas-ipc — CHANGES_REQUESTED** [review]
See earlier comments.

**Review by Eggflaw — COMMENTED** [review]
Fixed.

**Review by jrelvas-ipc — APPROVED** [review]
All looks fine to me!

**Pull request merged**